### PR TITLE
Enabling 535 era + other things

### PR DESCRIPTION
**Pull request by IhateTrains**
- enabled the Gothic Wars era
- updated barony name in building potential
- added basic k_lithuania to New World Kingdoms
- Validator fixes
- changed fall_of_rome penalty ID to fall_of_rome_MA to avoid confusion with global flag
 EDIT: changed to fall_of_rome_moral_authority
- changed picture and title for 2000th anniversary of the Gothic kingdom
- event_picture sprites fixes
- code cleaning
- fixed the Senate Approval firing when d_senate has no holder
- added some Wikipedia links
- historical dates of death of Euric and Athalaric
- added missing nick_the_dismal

### CODE REVIEW

File: WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt [hunk 1]
```diff
@@ -2812,6 +2812,12 @@ k_poland = {
 	color={ 150 23 23 }
 	color2={ 200 0 0 }
 }
+k_lithuania = {
+	color = { 75 125 6 }
+	color2 = { 255 255 255 }
+	
+	capital = 420 # Aukshayts/Aukszota
+}
```
Comment (loup99) [inline WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt]: Why did you add this kingdom?

File: WTWSMS/common/religion_modifiers/WtWSMS_religion_modifiers.txt [hunk 2]
```diff
@@ -14,12 +14,12 @@ affected_by_pestilence = {
 	years = 5
 }
 
-fall_of_rome= {
+fall_of_rome_MA = {
```
Comment (loup99) [inline WTWSMS/common/religion_modifiers/WtWSMS_religion_modifiers.txt]: Maybe "fall_of_rome_moral_authority" would be more explicit? Just my preference though.

File: WTWSMS/history/characters/kannada.txt [hunk 3]
```diff
@@ -111,7 +111,7 @@
 }
 
 74404 = {
-	name="Flavius Julius" 
+	name="Majorian"
```
Comment (loup99) [inline WTWSMS/history/characters/kannada.txt]: Why did you rename him? Flavius Julius Valerius Majorianus would work, but just Majorian poses a problem in French and Spanish.

File: WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt (hunk 1 above)
Comment (IhateTrains) [reply WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt]: It was missing and similar kingdoms (Pomerania, Poland etc.) are also here

File: WTWSMS/common/religion_modifiers/WtWSMS_religion_modifiers.txt (hunk 2 above)
Comment (IhateTrains) [reply WTWSMS/common/religion_modifiers/WtWSMS_religion_modifiers.txt]: ok

File: WTWSMS/history/characters/kannada.txt (hunk 3 above)
Comment (IhateTrains) [reply WTWSMS/history/characters/kannada.txt]: He isn't known as "Flavius Julius". I will change it to his simple Latin name, Maiorianus.

File: WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt (hunk 1 above)
Comment (loup99) [reply WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt]: Missing from what? Is there a Lithuania in vanilla?

File: WTWSMS/history/characters/kannada.txt (hunk 3 above)
Comment (loup99) [reply WTWSMS/history/characters/kannada.txt]: Okay.

File: WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt (hunk 1 above)
Comment (IhateTrains) [reply WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt]: It should also be possible to form this kingdom in later game

Comment (IhateTrains) [reply WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt]: I think I took this from vanilla but I'm not sure, I will check

Comment (IhateTrains) [reply WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt]: Yes, there is

Comment (loup99) [reply WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt]: Okay.

File: WTWSMS/common/religion_modifiers/WtWSMS_religion_modifiers.txt (hunk 2 above)
Comment (IhateTrains) [reply WTWSMS/common/religion_modifiers/WtWSMS_religion_modifiers.txt]: no, wait. I forgot to edit this file

Comment (IhateTrains) [reply WTWSMS/common/religion_modifiers/WtWSMS_religion_modifiers.txt]: Now really done
